feature_tests/kotlin/somelib/src/main/kotlin/dev/diplomattest/somelib/CallbackWrapper.kt

```diff
@@ -39,7 +39,11 @@ internal class DiplomatCallback_CallbackWrapper_test_multi_arg_callback_diplomat
                 }
             }
     @JvmField
-    internal var destructor: Pointer = Pointer(0L);
```

Comment:
issue: I can't leave a comment on a binary file, but we shouldn't check in libsomelib.so. Add a .gitignore if it's necessary to make git happy.

Reply:
oh woops! sorry about that, it's gone